docs/docs/querying-with-graphql.md

```diff
@@ -1,71 +1,179 @@
 ---
-title: Querying with GraphQL
+title: Querying data with GraphQL
 ---
 
-> This page is a work in progress.
+There are many options for loading data into React components. One of the most
+popular and powerful of these is a technology called
+[GraphQL](http://graphql.org/).
+
+GraphQL was invented at Facebook to help product engineers _pull_ needed data into
+React components.
+
+GraphQL is a **q**uery **l**anguage (the _QL_ part of its name). If you're
+familiar with SQL, it works in a very similar way. Using a special syntax, you describe
+the data you want in your component and then that data is given
+to you.
+
+Gatsby uses GraphQL to enable [page and layout
+components](/docs/building-with-components/) to declare what data they and their
+sub-components need. Gatsby then handles making sure that data is available in
+the browser when needed by your components.
+
+## What does a GraphQL query look like?
+
+GraphQL lets you ask for the exact data you need. Queries look like JSON:
+
+```graphql
+{
+  site {
+    siteMetadata {
+      title
+    }
+  }
+}
+```
+
+Which returns:
+
+```json
+{
+  "site": {
+    "siteMetadata": {
+      "title": "A Gatsby site!"
+    }
+  }
+}
+```
+
+A basic page component with a GraphQL query might look like this:
+
+```jsx
+import React from "react";
+
+export default ({ data }) => (
+  <div>
+    <h1>About {data.site.siteMetadata.title}</h1>
+    <p>We're a very cool website you should return to often.</p>
+  </div>
+);
+
+export const query = graphql`
+  query AboutQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
+`;
+```
+
+The result of the query is automatically inserted into your React component on the `data` prop. GraphQL and Gatsby lets ask for data and then immediately start using it.
```

Comment:
Missing a 'you' here? `GraphQL and Gatsby lets _you_ ask for data`.

Reply:
👍